### PR TITLE
re: Document `EdgeVerifierActor` pt1

### PR DESCRIPTION
**Pull request by pmnoxx**
Document methods as requires in https://github.com/near/nearcore/pull/5230#discussion_r754603985 https://github.com/near/nearcore/pull/5230#discussion_r754604357

### CODE REVIEW

File: chain/network/src/peer_manager/peer_manager_actor.rs [hunk 1]
```diff
@@ -1209,13 +1209,22 @@ impl PeerManagerActor {
         );
     }
 
-    fn verify_edges(&mut self, ctx: &mut Context<Self>, peer_id: PeerId, edges: Vec<Edge>) {
+    /// Sends list of edges to check their signatures to `EdgeVerifierActor`.
+    /// Bans peers if an invalid edge is found.
```
Comment (mm-near) [inline chain/network/src/peer_manager/peer_manager_actor.rs:1213]: so these are the edges that we've received from this peer_id, right?

And we'll ban this peer (singular) - if signature on any of the edges is invalid.

File: chain/network/src/peer_manager/peer_manager_actor.rs [hunk 2]
```diff
@@ -1569,8 +1578,8 @@ impl Actor for PeerManagerActor {
         // Start active peer stats querying.
         self.monitor_peer_stats_trigger(ctx);
 
-        // Read verified edges and broadcast them.
-        self.broadcast_edges_trigger(ctx);
+        // Read valid edges from `EdgesVerifierActor` and broadcast.
```
Comment (mm-near) [inline chain/network/src/peer_manager/peer_manager_actor.rs:1581]: I'd also add 'Periodically'

File: chain/network/src/routing/edge_verifier_actor.rs [hunk 3]
```diff
@@ -21,11 +21,17 @@ impl Handler<StopMsg> for EdgeVerifierActor {
     }
 }
 
-impl Handler<EdgeList> for EdgeVerifierActor {
+/// EdgeListToVerify contains list of Edges, and it's associated with a connected peer.
```
Comment (mm-near) [inline chain/network/src/routing/edge_verifier_actor.rs:24]: nit: here you say 'ToVerify', but code says 'ToValidate'

File: chain/network/src/routing/edge_verifier_actor.rs [hunk 4]
```diff
@@ -21,11 +21,17 @@ impl Handler<StopMsg> for EdgeVerifierActor {
...
+/// Check signatures of all edges in `EdgeListToVerify` and if some signature is not valid,
```
Comment (mm-near) [inline chain/network/src/routing/edge_verifier_actor.rs:25]: s/some/any

**Review by mm-near — APPROVED** [review]

File: chain/network/src/peer_manager/peer_manager_actor.rs (hunk 1 above)
Comment (pmnoxx) [reply chain/network/src/peer_manager/peer_manager_actor.rs:1213]: Yes, I updated the documentation to make it clear.

File: chain/network/src/routing/edge_verifier_actor.rs (hunk 3 above)
Comment (pmnoxx) [reply chain/network/src/routing/edge_verifier_actor.rs:24]: Fixed.

File: chain/network/src/routing/edge_verifier_actor.rs (hunk 4 above)
Comment (pmnoxx) [reply chain/network/src/routing/edge_verifier_actor.rs:25]: Thanks, fixed.

File: chain/network/src/types.rs [hunk 5]
```diff
@@ -839,7 +839,7 @@ pub enum NetworkRequests {
     },
 }
 
-pub struct EdgeList {
+pub struct EdgeListToValidate {
```
Comment (matklad) [inline chain/network/src/types.rs:842]: I'd probably name actor requests as verbs: `Handle<ValidateEdgeList>`

**Review by matklad — APPROVED** [review]

**Pull request merged**